pkg/provisioner/ironic/update_opts.go

```diff
@@ -88,6 +88,7 @@ func sanitisedValue(data interface{}) interface{} {
 		if strings.Contains(k.String(), "password") {
 			safeDatumValue = reflect.ValueOf("<redacted>")
 		}
+		safeDatumValue = reflect.ValueOf(redactedProbableURL(safeDatumValue.Interface()))
```

Comment:
... I'd rather have the reflection concentrated in this function.

Reply:
Will do.